src/sagemaker/jumpstart/estimator.py

```diff
@@ -39,128 +33,78 @@ class JumpStartEstimator(Estimator):
-        self.model_version = model_version
-        self.kwargs_for_base_estimator_class = deepcopy(kwargs_for_base_estimator_class)
-
-        self.kwargs_for_base_estimator_class = update_dict_if_key_not_present(
-            self.kwargs_for_base_estimator_class,
-            "image_uri",
-            image_uris.retrieve(
-                region=None,
-                framework=None,
-                image_scope="training",
-                model_id=model_id,
-                model_version=model_version,
-                instance_type=self.instance_type,
-            ),
-        )
-
-        self.kwargs_for_base_estimator_class = update_dict_if_key_not_present(
-            self.kwargs_for_base_estimator_class,
-            "model_uri",
-            model_uris.retrieve(
-                script_scope=JumpStartScriptScope.TRAINING,
-                model_id=model_id,
-                model_version=model_version,
-            ),
-        )
-
-        self.kwargs_for_base_estimator_class = update_dict_if_key_not_present(
-            self.kwargs_for_base_estimator_class,
-            "script_uri",
-            script_uris.retrieve(
-                script_scope=JumpStartScriptScope.TRAINING,
-                model_id=model_id,
-                model_version=model_version,
-            ),
-        )
-
-        default_hyperparameters = hyperparameters.retrieve_default(
-            region=region, model_id=model_id, model_version=model_version
+        estimator_init_kwargs = get_init_kwargs(
+            model_id=model_id,
+            model_version=model_version,
+            instance_type=instance_type,
+            instance_count=instance_count,
+            region=region,
+            image_uri=image_uri,
+            model_uri=model_uri,
+            source_dir=source_dir,
+            entry_point=entry_point,
+            hyperparameters=hyperparameters,
+            metric_definitions=metric_definitions,
+            kwargs=kwargs,
         )
 
-        curr_hyperparameters = self.kwargs_for_base_estimator_class.get("hyperparameters", {})
-        new_hyperparameters = deepcopy(curr_hyperparameters)
-
-        for key, value in default_hyperparameters:
-            new_hyperparameters = update_dict_if_key_not_present(
-                new_hyperparameters,
-                key,
-                value,
-            )
+        self.model_id = estimator_init_kwargs.model_id
+        self.model_version = estimator_init_kwargs.model_version
+        self.instance_type = estimator_init_kwargs.instance_type
+        self.instance_count = estimator_init_kwargs.instance_count
+        self.region = estimator_init_kwargs.region
 
-        if new_hyperparameters == {}:
-            new_hyperparameters = None
+        super(JumpStartEstimator, self).__init__(**estimator_init_kwargs.to_kwargs_dict())
 
-        self.kwargs_for_base_estimator_class["hyperparameters"] = new_hyperparameters
+    def fit(self, *largs, **kwargs) -> None:
```

Comment:
please use all the arguments of [fit](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/estimator.py#L1128)

At least keep `inputs` and `wait` args, but preferably also `logs` and `job_name`